blocks/library/freeform/buttons/simple.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { find } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from 'element';
+import { Button, TinymceIcon } from 'components';
+
+class SimpleButton extends Component {
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			isActive: false,
+			isDisabled: false,
+		};
+		this.mounted = true;
+	}
+
+	componentDidMount() {
+		const { button } = this.props;
+		const fnNames = [ 'onPostRender', 'onpostrender', 'OnPostRender' ];
+		const onPostRender = find( fnNames, ( fn ) => button.hasOwnProperty( fn ) );
+		if ( onPostRender ) {
+			button[ onPostRender ].call( {
+				active: ( isActive ) => this.mounted && this.setState( { isActive } ),
+				disabled: ( isDisabled ) => this.mounted && this.setState( { isDisabled } ),
```

Comment:
Interesting! I will dig into the TinyMCE code and see if I can figure out what's going on. It will probably turn out that the 'this' object and the 'event.control' object are the same thing and that different buttons functions access them by different routes depending on the implementor.